spec/predicates/test-results.md

```diff
@@ -0,0 +1,148 @@
+# Predicate type: Test Results
+
+Type URI: https://in-toto.io/attestation/test-result/v0.1
+
+Version: 0.1.0
+
+Authors:
+
+## Purpose
+
+This predicate type defines a generic schema to express the result of running
+tests in software supply chains. The schema may be extended to support different
+types of testing or specific test harnesses.
+
+## Use Cases
```

Comment:
Out of curiosity, do you have immediate use-cases in mind?

Reply:
I'm working with an adopter to deploy in-toto and I started sketching this out based on their test framework.